app/models/tool.rb

```diff
@@ -37,95 +37,110 @@ class Tool < ApplicationRecord
     title: 'Name',
```

Comment:
**IMPORTANT**

Does this need changing? Jenn wrote a comment [here](https://unep-wcmc.codebasehq.com/projects/tradehub-navigator/tickets/44#update-70161021) which says 

> [@maxime-seiter](https://unep-wcmc.codebasehq.com/users/1c64d8de-fced-1c62-7bdb-b74ad1b5e182) It should be Initiative, not name.

Does that not mean that we need to edit line 37 which currently still has 'Name'?

Reply:
Yes but I have done it in PR#29.